rules/S4347/metadata.json

```
@@ -1,5 +1,5 @@
 {
-  "title": "\"SecureRandom\" seeds should not be predictable",
+  "title": "Secure random number generators must not output predictable values",
```

```suggestion
  "title": "Secure random number generators should not output predictable values",
```
Typically "should" is used over "must" in titles of vulnerabilities